metaflow/plugins/aip/exit_handler_decorator.py

```diff
@@ -39,7 +39,7 @@ def wrapped(*args, **kwargs):
 
 def exit_handler_retry(
     times: int = EXIT_HANDLER_RETRY_COUNT,
-    minutes_between_retries: int = BACKOFF_DURATION_INT,
+    minutes_between_retries: int = int(BACKOFF_DURATION_INT),
```

Comment:
Would it be better to fix it in `metaflow.plugins.aip.aip_constants`?

Reply:
much better! fixed!